GNUmakefile

```diff
@@ -174,11 +195,19 @@ ifeq ($(CIRCLE_BRANCH), main)
 	@docker push $(CI_DEV_DOCKER_NAMESPACE)/$(CI_DEV_DOCKER_IMAGE_NAME):latest
 endif
 
-# linux builds a linux binary independent of the source platform
+# linux builds a linux binary compatible with the source platform
 linux:
+	@mkdir -p ./pkg/bin/linux_$(GOARCH)
+	CGO_ENABLED=0 GOOS=linux GOARCH=$(GOARCH) go build -o ./pkg/bin/linux_$(GOARCH) -ldflags "$(GOLDFLAGS)" -tags "$(GOTAGS)"
+
+linux-amd64:
 	@mkdir -p ./pkg/bin/linux_amd64
 	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o ./pkg/bin/linux_amd64 -ldflags "$(GOLDFLAGS)" -tags "$(GOTAGS)"
 
+linux-arm64:
+	@mkdir -p ./pkg/bin/linux_arm64
+	CGO_ENABLED=0 GOOS=linux GOARCH=arm64 go build -o ./pkg/bin/linux_arm64 -ldflags "$(GOLDFLAGS)" -tags "$(GOTAGS)"
+
```

Comment:
why not combine those 3 and have the same build command wit different `GOARCH` values?

Reply:
The honest answer is I wasn't sure how to programmatically exec make targets with different values of `GOARCH`, and it wasn't something I could find at the top of StackOverflow. Maybe you know the secret sauce? :D

Reply:
Figured it out. Will update 🙇